tests/unit/test_cli.py

```
@@ -0,0 +1,166 @@
+from typer.testing import CliRunner
+
+from potpie import cli
+
+pytestmark = pytest.mark.unit
+
+runner = CliRunner()
+
+
+def _make_git_repo(path: Path) -> Path:
+    path.mkdir()
+    subprocess.run(["git", "init", "-b", "main"], cwd=path, check=True)
```

Ruff (0.15.12)

[error] Starting a process with a partial executable path

(S607)